files/start-strongswan.sh

```diff
@@ -48,6 +50,15 @@ then
         exit 0
     }
 
+    _set_default_variables() {
+        : ${IPSEC_REMOTE_IP:=%any}
+        : ${IPSEC_LOCALIP:=%any}
```

Comment:
either split IP with a _ or do not. (mixing both with IPSEC_REMOTE_IP and IPSEC_LOCALIP)